.github/actions/security-scans/action.yml

```
@@ -1,9 +1,24 @@
 name: "Security Scans"
 description: "Runs security scans on the app"
+inputs:
+  github-requirements-branch:
+    description: 'Branch of the github requirements repository to check out'
+    required: false
+    default: 'main'
 runs:
   using: "composite"
   steps:
+    - name: Install Requirements
+      run: pip install -r ${{ github.action_path }}/requirements.txt
+      shell: bash
+
     - name: Run Security Scans
-      run: |
-        echo "Running security scans"
+      run: 
+      - set -e
+      - |
+        python ${{ github.action_path }}/run_gitlab_pipeline.py 'GitHub Requirements' ${{ inputs.github-requirements-branch }} -t 600 -v "{
```

should this be more than 600 seconds for timeout? I feel like this fails sometimes when the scans pass but take longer

---

Yeah let's bump this
```suggestion
        python ${{ github.action_path }}/run_gitlab_pipeline.py 'GitHub Requirements' ${{ inputs.github-requirements-branch }} -t 1200 -v "{
```